accounts/views.py

```diff
@@ -1284,7 +1284,21 @@ def upload(request, no_flash=False):
             form = UploadFileForm(request.POST, request.FILES)
             if form.is_valid():
                 submitted_files = request.FILES.getlist('files')
+                if not submitted_files: 
+                    #this allows upload_tests to work, requests use the dict key in singular
```

Comment:
Hmmmm, this comment is a bit strange :S Could this be because in line 73 of test_upload.py you set "file" in singular?